go-controller/pkg/ovn/policy_test.go

```diff
@@ -80,9 +69,17 @@ func newNetworkPolicy(name, namespace string, podSelector metav1.LabelSelector,
 	return policy
 }
 
+func getFakeBaseController(netInfo util.NetInfo, netConfInfo util.NetConfInfo) *BaseNetworkController {
+	return &BaseNetworkController{
+		controllerName: netInfo.GetNetworkName() + "-network-controller",
+		NetInfo:        netInfo,
+		NetConfInfo:    netConfInfo,
+	}
+}
+
 func getDefaultDenyData(networkPolicy *knet.NetworkPolicy, ports []string,
```

Comment:
If we merge https://github.com/ovn-org/ovn-kubernetes/pull/3540 soon, we will be able to change the signature of this function, then pass default network args for existing wrappers, and create new functions like `getDefaultDenyDataMultinet` and `getPolicyDataMultinet`.
In that case calls in this file won't need to be changed